README.md

```diff
@@ -6,5 +6,19 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+
+  Rails is a web framework to help build the back end of web apps. It provides default structures for a database, web service and web pages.
+
+
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
+  Image inserted in submissions directory.
```

Comment:
The image is right up to the model; however, after that point it is mistaken. The controller is the glue - it calls methods on the model, takes the returned results, and uses them to build a JSON string as a response.